tee-worker/cli/src/benchmark/mod.rs

```diff
@@ -145,9 +146,11 @@ impl BenchmarkCommand {
 			let account = get_pair_from_str(trusted_args, a.public().to_string().as_str());
 			let initial_balance = 10000000;
 
+			let funding_identity = Identity::Substrate(funding_account_keys.public().into());
```

Comment:
I see such conversion in variant places, is it worth having a `From` impl that does `Address32` or `[u8; 32]` to `Identity` conversion? Maybe only `[u8; 32]` is enough because of the `into()` chain

Same for `[u8; 20]`